prime-router/src/test/kotlin/fhirengine/azure/FhirFunctionIntegrationTests.kt

```diff
@@ -727,679 +721,6 @@ class FhirFunctionIntegrationTests() {
         }
     }
 
-    @Test
-    fun `test successfully processes a convert message for a bulk (ndjson) FHIR message`() {
```

Comment:
Are the test cases being deleted here covered somewhere else now?

Reply:
Yep, these are covered in FhirConverterIntegrationTests